{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "210cd2a9_150fc34e",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 17
      },
      "lineNbr": 0,
      "author": {
        "id": 1000340
      },
      "writtenOn": "2021-09-21T21:30:16Z",
      "side": 1,
      "message": "l1 test passed when I ran the failing test by itself, so trying l2 again.",
      "revId": "3ea9ff249a9d5ef2aa97ab3aa2d3f50905333778",
      "serverId": "8f6f209b-db1a-4cbf-aa44-c8bc30e9bfda"
    }
  ]
}